serverless/serverless-overview.adoc

```diff
@@ -0,0 +1,27 @@
+[id="serverless-overview"]
+= Red Hat OpenShift Serverless overview
+include::modules/common-attributes.adoc[]
+:context: serverless-overview
+
+toc::[]
+
+[IMPORTANT]
```

Comment:
It's similar to the language in the [tech preview module](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#indicating-technology-preview-features). Where did you source this note? I'm curious if the tech preview module's fallen out of date.

Reply:
Fixed, pinged you on Slack re the source.